From 5e2cedb10b77766db1a2f2a51167e91245d27de0 Mon Sep 17 00:00:00 2001
From: Dexter Gaon-Shatford <dexter@gaonshatford.ca>
Date: Sat, 22 Oct 2022 10:56:36 -0400
Subject: [PATCH] Fix default configuration for Logitech F310

The default configuration for the Logitech F310 gamepad is incorrect in
the 2.5.9 release. This patch applies the (correct) configuration from
the project upstream.
---
 data/InputAutoCfg.ini | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/InputAutoCfg.ini b/data/InputAutoCfg.ini
index 4db5576..368a761 100644
--- a/source/mupen64plus-input-sdl/data/InputAutoCfg.ini
+++ b/source/mupen64plus-input-sdl/data/InputAutoCfg.ini
@@ -597,15 +597,15 @@ DPad L = hat(0 Left)
 DPad D = hat(0 Down)
 DPad U = hat(0 Up)
 Start = button(7)
-Z Trig = button(5)
+Z Trig = axis(5+)
 B Button = button(2)
 A Button = button(0)
 C Button R = axis(3+)
 C Button L = axis(3-)
 C Button D = axis(4+)
 C Button U = axis(4-)
-R Trig = axis(5-)
-L Trig = axis(2-)
+R Trig = button(5)
+L Trig = button(4)
 Mempak switch = button(1)
 Rumblepak switch = button(3)
 X Axis = axis(0-,0+)
-- 
2.38.1

